index.html

```diff
@@ -3,3 +3,6 @@ <h1>Git tutorial</h1>
 git diff.
 git commit -- amend.
 git merge
+conflict
+git confilicti2
+git_pull_request.
```

Comment:
Could you delete line#5.
```suggestion
git_pull_request.
```
```suggestion
git_pull_request.
```